core/revised/rogue.tex

```diff
@@ -36,7 +36,7 @@
     \crystal{water}{12pt} & %
     \tspec{Arcane Trickster}: Requires Water level 10. Increase your MP bonus multiplier by 1 at all levels. In addition, you gain one of the following Spell groups: \tspellgroup{Transform}, \tspellgroup{Poison}, \tspellgroup{Images}, \tspellgroup{Teleport}, \tspellgroup{Weaken}, or \tspellgroup{Slow}. \\
     \crystal{fire}{12pt} & %
-    \tspec{Dismantle}: Requires Fire level 10. You gain the Quick magic action \taction{Dismantle}. You use your mastery over machines and technology to disarm threats. Perform a Fire vs Earth attack, difficulty 70, against a target. If the target is a \tmtypeconstr{} (such as a golem, machine, or robot), it is instantly reduced to 0 HP\@. Treat this effect as a \tstatus{Fatal} type status. \\
+    \tspec{Dismantle}: Requires Fire level 10. You gain the Quick magic action \taction{Dismantle}. You use your mastery over machines and technology to disarm threats. Perform a Fire vs Earth attack, difficulty 70, against a target. If the target is a \tmtypeconstr{} (such as a golem, machine, or robot), inflict the \tstatus{Death} status\@. If the target is not a \tmtypeconstr{}, remove one positive status effect from it and disable the corresponding permanent status effect, if any, until the end of next round. \\
```

Comment:
\@ is only needed if the item before it can be construed as an abbreviation by LaTeX.  HP\@. is appropriate, status\@. is not -- though it doesn't hurt.

Reply:
5e577d